vmdb/app/models/ems_microsoft.rb

```diff
@@ -96,7 +96,8 @@ def execute_power_operation(cmdlet, vm_uid_ems, *parameters)
     return unless vm_uid_ems.guid?
 
     params  = parameters.join(" ")
-    command = "powershell #{cmdlet}-SCVirtualMachine -VM (Get-SCVirtualMachine -ID #{vm_uid_ems}) #{params}"
+    command = "powershell Import-Module VirtualMachineManager; Get-VMMServer #{ipaddress}; "
```

Comment:
@jrafanie Get-VMMServer does work with either hostnames or ip address.
I just tested it with 
"localhost" - success
"127.0.0.1" - success
"::1" - failure
"[::1]" - success.

So apparently IPv6 addresses need to be wrapped in brackets for (at least some Powershell/SCVMM commands).